plugins/fcp_can_c/fcp_can_c/generator.py

```diff
@@ -123,3 +124,59 @@ def check_impl_size(
                 )
             else:
                 return Ok(())
+
+        @register(verifier, "struct")  # type: ignore
```

Comment:
Move the new checks to https://github.com/joajfreitas/fcp-core/blob/master/src/fcp/verifier.py